rmw_zenoh_cpp/src/detail/rmw_subscription_data.hpp

```diff
@@ -132,6 +132,10 @@ class SubscriptionData final
     const void * type_support_impl,
     std::unique_ptr<MessageTypeSupport> type_support);
 
+  bool init();
+
+  bool initted_{false};
```

Comment:
Can we
- call this `initialized_` instead
- move it's definition above `bool is_shutdown_`
- initialize the value to `false` in the constructor initializer list

Reply:
Done in efd8918.